dashboard/config/libraries/variableLog.interpreted.js

```diff
@@ -27,52 +44,71 @@ function updateLog(){
   }
 
 }
+*/
 
-function varWatchers(varLog){
-  for(var key in varLog){
+function varWatchers(varLog) {
```

Comment:
It would be great to have a comment describing what this does.  Also, it seems slightly odd that a function called `varWatchers` would then do some drawing?